app/components/SearchTableLayout.tsx

```diff
@@ -64,7 +64,6 @@ export const SearchTableLayoutComponent: React.FunctionComponent<Props> = (
               <th className="master-select">
                 <label>
                   Select All
-                  <br />
                   <input
```

Comment:
`input` should be a sibling of `label` (you're going to need an `id` and `htmlFor` for accessibility, which would allow you to use a flexbox for `master-select` (I see it's already a flexbox for some reason, it just needs to have `flex-direction: row`.

Reply:
There's no expectation that `<input />` be a sibling of `<label>` instead of a child ([reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)). The purposes of making it a child are:
- clicking a `<label>` element natively causes the associated input to be checked/unchecked without the use of any custom/duplicate event listeners.
- avoids the need for specifying excess ids to associate the label with the input, as the parent-child relationship automatically associates it.

This PR only needs to change CSS positioning, but I had to remove the `<br>` which is no longer considered good practise anyway (versus using CSS for spacing and positioning).